scripts/update-third-party-notices-pr.sh

```diff
@@ -10,7 +10,17 @@
 GIT_REMOTE_NAME="origin"
 BASE_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 PR_BRANCH="update-third-party-notices-${BASE_BRANCH}"
-COMMIT_MESSAGE="[${BASE_BRANCH}] chore: update third party notices [ci skip]"
+
+# Don't use [ci skip] for branches other than main, since branch protection rules* will require CI
+# checks to pass before merging. If [ci skip] is used, we will need to manually push an empty commit
+# without [ci skip] to cause the CI checks to run.
```

Comment:
Example of manual empty commit https://github.com/confluentinc/vscode/pull/2361/commits/d2138dfa5d138dcaa8dc7bc41ef18d9a5d30b9e8, done with `git commit --allow-empty -m "empty commit for CI"`